tests/test_integration.py

```diff
@@ -18,12 +18,30 @@
 import unittest
 import uuid
 from unittest.mock import call, MagicMock, patch, PropertyMock
+from parameterized import parameterized
+from itertools import product
 
 import aepsych.server as server
 import aepsych.utils_logging as utils_logging
 import torch
 from aepsych.config import Config
 
+params = {'singleStimuli':{
+    'x1':[0.1, 0.2, 0.3, 1, 2, 3, 4],
+    'x2':[4, 0.1, 3, 0.2, 2, 1, 0.3]
+}, 'multiStimuli':{
+    'x1': [[0.1, 0.2], [0.3, 1], [2, 3], [4, 0.1], [0.2, 2], [1, 0.3], [0.3, 0.1]],
+    'x2': [[4, 0.1], [3, 0.2], [2, 1], [0.3, 0.2], [2, 0.3], [1, 0.1], [0.3, 4]]
+}}
+
+outcomes = {'singleOutcome':[1, -1, 0.1, 0, -0.1, 0, 0],
+            'multiOutcome':[[1, 0], [-1, 0], [0.1, 0], [0, 0], [-0.1, 0], [0, 0], [0, 0]]}
+
+all_tests = ['singleStimuli_singleOutcome',
+             'multiStimuli_singleOutcome', 'multiStimuli_multiOutcome']
+#! This should be supported! Uncoment when ready
+# all_tests.append('singleStimuli_multiOutcome')
```

Comment:
This test case did not pass. I commented this test case because I think multi-outcome it's still not suported🤔